### PR TITLE
OAuth2 flow for instructor

### LINKED ISSUE #194: OAuth2 flow for instructor

### PR DESCRIPTION
**Pull request by vutoan1245**
Resolve #194

### CODE REVIEW

**Review by ljoks — APPROVED** [review]

**Review by kennygperez — CHANGES_REQUESTED** [review]
Added new table `Lti` to drop/init flow in the `db/mysql.ts` handler

File: backend/src/handlers/auth/lti.ts [hunk 1]
```diff
@@ -4,20 +4,114 @@ import {
...
+			where: {
+				canvasUserId: Number(userId),
+			},
+		});
+		await Lti.create({
+			canvasUserId: Number(userId),
+			canvasCourseId: Number(courseId),
+		});
+
+		const scopes = [
```
Comment (kennygperez) [inline backend/src/handlers/auth/lti.ts:44]: Awesome stuff here! I need todo this for the mobile oauth

**Comment by vutoan1245** [comment]
@kennygperez I  added it in the `models` array.

**Comment by kennygperez** [comment]
Ah you're right, my mistake

**Review by kennygperez — APPROVED** [review]

**Pull request merged**